cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/TransactionHistoryPage.java

```diff
@@ -259,14 +259,37 @@ public String setCategory(String category) {
      * @return the text of the first row
      */
     public String confirmCreation() {
```

Comment:
Low impact: The 2nd, 3rd, and 4th if statements will never be run.
Suggestion: Reverse the order of the if statements or remove the unnecessary ones.